pmm-app/src/pmm-settings/components/MetricsResolution/MetricsResolution.tsx

```diff
@@ -40,107 +40,104 @@ export const MetricsResolution: FC<MetricsResolutionProps> = ({ metricsResolutio
     },
     tooltipLinkText,
   } = Messages;
-  const onChangeInput = (key: string, value: string) => {
-    updateResolutions({ ...updatedResolutions, [key]: value });
-    updateCustomResolutions({ ...customResolutions, [key]: value });
+  const changeResolutions = (state: any, changeValue: any, newResolutions: MetricsResolutions) => {
+    Object.keys(newResolutions).forEach(
```

Comment:
I suggest to use Object.entries. Looks like it's more appropriate here.

Reply:
Done 👍